project_scrum/views/project_project.xml

```diff
@@ -29,11 +28,38 @@
             </field>
         </field>
     </record>
- 
+
+    <record id="act_project_tasks" model="ir.actions.act_window">
+        <field name="name">Tasks</field>
+        <field name="res_model">project.task</field>
+        <field name="view_mode">kanban,tree,form,calendar,pivot,graph</field>
+        <field name="context">{
+            'group_by': 'stage_id',
+            'search_default_project_id': [active_id],
+            'default_project_id': active_id,
```

Comment:
default_{fieldname}  will then give you the default in your call, therefore making project_id the default in the project.task view you will open.
beware, this means that this action can be called only from a project form (as the button does). Having it elsewhere will cause obvious problems. "active_id" on correct model  needs to be managed by you , add a word of comment on this.

Reply:
This action was copied as it is from `act_project_project_2_project_task_all` minus the changes on the views. This action is only intended to be used only on the button, but of course can be reused anywhere they want.

If someone wants to use it, they have to read it first and when they do they should know what the `active_id` stands for and any further commentary will confuse or state the obvious.